test/test_torch.py

```diff
@@ -5199,6 +5199,15 @@ def test_lazy_clone_view_materialize(self, device, dtype):
         self.assertTrue(torch._C._data_address(view) == t_new_data_addr)
         self.assertTrue(torch._C._data_address(clone) == orig_data_ptr)
 
+    @skipXLA
+    @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
+    def test_lazy_clone_binary_op_no_materialize(self, device, dtype):
+        t = torch.tensor([[0, 1], [2, 3]], device=device, dtype=dtype)
+        clone = t._lazy_clone()
+        res = t + clone
+        self.assertTrue(torch._C._is_cow_tensor(t))
+        self.assertTrue(torch._C._is_cow_tensor(clone))
```

Comment:
Would it make sense to add something to the OpInfos tests to check this for all the operations, or would that be overkill?

Reply:
Meh, I'd only do it if you think there is something legitimate that you'd expect it to catch

Reply:
Upon further reflection, I think this would be useful for keeping track of how many pointwise kernels you've ported to mark const inputs.

Reply:
Sounds good, I'll look into that in a follow up